docs/source/conf.py

```diff
@@ -21,7 +21,7 @@
 copyright = '2021, Labelbox'
 author = 'Labelbox'
 
-release = '3.61.0'
+release = 'labelbox_3.61.1'
```

Comment:
Should be only `3.61.1`, remove `labelbox_` prefix

Reply:
Done.